tests/foreman/cli/test_contentviews.py

```diff
@@ -1004,8 +929,10 @@ def test_cv_promote_custom_content(self):
             u'id': result.stdout['versions'][0]['id'],
             u'to-lifecycle-environment-id': self.env1['id'],
         })
-        self.assertEqual(result.return_code, 0,
-                         "Promoting a version of CV was not successful")
+        self.assertEqual(
+            result.return_code, 0,
+            "Promoting a version of CV was not successful"
```

Comment:
You have removed the third parameter, the error message that gets displayed to the user, in other parts of the code, so perhaps you should remove this one here too?